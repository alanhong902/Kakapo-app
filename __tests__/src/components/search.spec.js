import React from 'react';
import { Search } from 'components/import/search';
import { getData, createComponentWithIntl } from '../helper';

test('<Search/> render', () => {
  const props = {
    router: { location: { pathname: '/youtube' } },
    ...getData('search'),
  };
  const tree = createComponentWithIntl(<Search {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
