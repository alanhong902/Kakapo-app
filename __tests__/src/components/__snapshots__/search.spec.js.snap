// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search/> render 1`] = `
<div
  className="youtube"
>
  <div
    className="group"
  >
    <input
      autoComplete="off"
      name="searchInput"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={true}
      type="text"
    />
    <label
      htmlFor="searchInput"
    >
      <span>
        Search YouTube (or enter a YouTube Link)
      </span>
    </label>
    <span
      className="highlight"
    />
    <span
      className="bar"
    />
  </div>
  <div
    className="youtube-items"
  />
</div>
`;
